source/includes/acts/get_api.md

```diff
@@ -207,7 +302,7 @@ Attribute | Type | Description
 --------- | ---- | -----------
 PageNumber | int | Page number of the results to fetch.
 PageSize | int | The number of results per page
-emissionTypeID | int | Emission Type ID is one of the type of emissions and is the one of the filter option provided for Emission Type
+emissionTypeIDs | int | Emission Type ID is one of the type of emissions and is the one of the filter option provided for Emission Type
```

Comment:
Emission Type ID is the unique identifier for an emission type record and is one of the filter options provided for Emission Type